### ISSUE TITLE
Language for serbo-croatian Wikipedia should not be `sh`

### ISSUE THREAD

**Issue by kelson42** [openzim/mwoffliner#2026]
See https://en.wikipedia.org/wiki/Serbo-Croatian and https://library.kiwix.org/#lang=sh

**Comment by sanks011** [comment]
@kelson42 I can try to fix it.. so should it be "hbs" or normal serbo-croatian? and anyother detail you wanna share like the file location

**Comment by kelson42** [comment]
@sanks011 Yes "hbs" should be OK, but actually we should maybe fix problem upstream at Wikipedia level? I will ping someone.

**Comment by benoit74** [comment]
We considered so far that we should open an issue on phabricator. 

But in fact, it looks like everything is "intended" and has been discussed since it is referenced on https://meta.wikimedia.org/wiki/Special_language_codes and does not have a corresponding phabricator task.

We will have to fix all these cases (not only serbo-croatian) with a mapping table on mwoffliner side.

**Comment by benoit74** [comment]
I'm not sure I want to fix this scraper side.

This looks like something very specific to Wikipedia, and scraper already has the `--customZimLanguage` setting. 

From my perspective, this should be fixed in recipe configuration.

In fact, this has already started to be done in Zimfarm, e.g. for map-bms, nds-nl and sh (sic): see https://github.com/openzim/zim-requests/issues/1430#issuecomment-3233585952 for details.

**Comment by benoit74** [comment]
Closing in favor of https://github.com/openzim/zim-requests/issues/1430, we won't do anything scraper-side.
